lib/active_storage/service/ik_file.rb

```diff
@@ -0,0 +1,63 @@
+module ImageKiIo
+  module ActiveStorage
+    class IKFile
+      attr_accessor :identifier
+
+      def initialize(identifier)
+        @identifier = identifier
+        @imagekit = ImageKitIo.client
+      end
+
+      def url
+        identifier['url']
+      end
+
+      def delete
+        begin
+          @imagekit.delete_file(file_id)
+        rescue
+          file_id
+        end
+      end
+
+      def file_type
+        identifier['fileType']
+      end
+
+      def thumbnail_url
+        identifier['thumbnailUrl']
+      end
+
+      def filename
+        identifier['name']
+      end
+
+      def size
+        identifier['size']
```

Comment:
There are more parameters in the file response, please check API reference - https://docs.imagekit.io/api-reference/upload-file-api/server-side-file-upload#understanding-response